coreneuron/nrniv/corenrn_parameters.h

```diff
@@ -33,7 +33,7 @@ THE POSSIBILITY OF SUCH DAMAGE.
 #include <fstream>
 #include <iostream>
 #include <iomanip>
-#include "coreneuron/utils/CLI11/CLI.hpp"
+#include "external/CLI11/include/CLI/CLI.hpp"
```

Comment:
Does the following work too?
```suggestion
#include "CLI/CLI.hpp"
```